os/arch/arm/src/armv7-a/arm_prefetchabort.c

```diff
@@ -65,6 +65,23 @@
  ****************************************************************************/
 
 extern uint32_t system_exception_location;
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: print_prefetchabort_detail
+ ****************************************************************************/
+
+static inline void print_prefetchabort_detail(uint32_t *regs, uint32_t ifar, uint32_t ifsr)
+{
+	_alert("#########################################################################\n");
+	_alert("PANIC!!! Prefetch Abort at instruction : 0x%08x\n",  regs[REG_PC]);
+	_alert("PC: %08x DFAR: %08x DFSR: %08x\n", regs[REG_PC], ifar, ifsr);
+	_alert("#########################################################################\n\n\n");
+}
```

Comment:
almost the same as print_dataabort_detail. 
Can we merge them? Including ARMv8-M's functions

Reply:
Could you let me know your opinion about this?